File: Example-iOS-ObjC/ViewController.m
#import "ViewController.h"
@import Backtrace;

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UITextView *textView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
}
- (IBAction) liveReportAction: (id) sender {
    NSArray *paths = @[@"/home/test.txt"];
    [[BacktraceClient shared] sendWithAttachmentPaths:paths completion:^(BacktraceResult * _Nonnull result) {
        NSLog(@"%@", result.message);
    }];
}

- (IBAction) crashAction: (id) sender {
    NSArray *array = @[];
    NSObject *o = array[1];
}


@end

File: Example-iOS/ViewController.swift
import UIKit
import Backtrace

class ViewController: UIViewController {
    @IBOutlet weak var textView: UITextView!

    override func viewDidLoad() {
        super.viewDidLoad()
        
    }

    @IBAction func liveReportAction(_ sender: Any) {
        BacktraceClient.shared?.send(attachmentPaths: [], completion: { (result) in
            print(result)
        })
    }

    @IBAction func crashAppAction(_ sender: Any) {
        let items = [String]()
        _ = items[1]
    }
}
